src/bin/writedisk.rs

```diff
@@ -113,7 +113,14 @@ fn choose_device() -> UsbBlockDevice {
     let mut input = String::new();
     io::stdin().read_line(&mut input).unwrap();
 
-    let index = input.trim().parse::<usize>().unwrap();
+    let index = match input.trim().parse::<usize>() {
+        Ok(i) => i,
+        Err(_e) => {
+            println!("invalid input. exiting");
```

Comment:
For consistency with the other error output, should just say "invalid input".

Reply:
Well, the other one says 'invalid index' so I figured it wasn't an issue, but I'll remove the exiting bit.